include/xtensor/xsort.hpp

```diff
@@ -266,30 +266,76 @@ namespace xt
         );
     }
 
+    /*****************************
+     * Implementation of argsort *
+     *****************************/
+
+    /**
+     * Sorting method.
+     *
+     * Predefined methods for performing indirect sorting.
+     *
+     * @see argsort(const xexpression<E>&, std::ptrdiff_t, sorting_method)
+     */
+    enum class sorting_method
+    {
+        /** Faster method but with no guarantee on preservation of order of equal elements
+         *  https://en.cppreference.com/w/cpp/algorithm/sort.
+         */
+        quick,
+        /** Slower method but with guarantee on preservation of order of equal elements
+         *  https://en.cppreference.com/w/cpp/algorithm/stable_sort.
+         */
```

Comment:
```suggestion
        /** 
         * Slower method but with guarantee on preservation of order of equal elements
         * https://en.cppreference.com/w/cpp/algorithm/stable_sort.
         */
```

Reply:
Done in 675a42700850b5e5ad6d47599329b4ccfc9def99